.storybook/themes/commerce.scss

```diff
@@ -285,6 +285,11 @@
 				),
 				video: (
 					margin: auto,
+					components: (
+						badge: (
+							align-self: baseline,
```

Comment:
Why wouldn't this be default for the badge?

Reply:
Moved